layouts/dashboard-layout.vue

```diff
@@ -0,0 +1,292 @@
+<script lang="ts" setup>
+import {
+  Dialog,
+  DialogPanel,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuItems,
+  TransitionChild,
+  TransitionRoot,
+} from '@headlessui/vue';
+import {
+  Bars3Icon,
+  ChatBubbleBottomCenterIcon,
+  Cog6ToothIcon,
+  IdentificationIcon,
+  XMarkIcon,
+} from '@heroicons/vue/24/solid';
+import { ChevronDownIcon } from '@heroicons/vue/20/solid';
+import { authStore } from '~/store/auth';
+import { useCardStore } from '~/store/cardStore';
+
+interface NavigationItem {
+  name: string;
+  href: string;
+  active: string[];
+  icon: any;
+}
+
+interface UserNavigationItem {
+  name: string;
+  href: string;
+}
+
+const route = useRoute();
+const { cardId } = storeToRefs(useCardStore());
+const { user } = authStore();
+const sidebarOpen = ref(false);
+
+const navigation: NavigationItem[] = [
+  { name: 'Cards', href: '/dashboard', active: ['/dashboard', '/cards/create', `/cards/${cardId.value}/edit`], icon: IdentificationIcon },
```

Comment:
you can rename the href to /cards.

Reply:
Done renaming route /dashboard to /cards